src/vstest.console/CommandLine/ITestCaseFilter.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.CommandLine
+{
+    using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// interface for test case filter
+    /// </summary>
+    internal interface ITestCaseFilter
```

Comment:
Suggest to remove the interface since it is only consumed in a deprecated scenario

Reply:
Ok